bin/ncp-update-nc

```diff
@@ -72,7 +74,7 @@ fi
 # https://github.com/nextcloud/server/issues/10949
 pgrep -cf cron.php &>/dev/null && { pkill -f cron.php; sleep 3; }
 pgrep -cf cron.php &>/dev/null && { echo "cron.php running. Abort"; exit 1; }
-mysql nextcloud <<<"UPDATE oc_jobs SET reserved_at=0;"
+mysql nextcloud <<<"UPDATE "$DB_FREFIX"jobs SET reserved_at=0;"
```

Comment:
Why are you closing and reopening the double quotes here? Did you intend to just quote the variable? Also, there's a typo in the variable's name.

```suggestion
mysql nextcloud <<<"UPDATE ${DB_PREFIX}jobs SET reserved_at=0;"
```

Reply:
IIRC I had initially `mysql nextcloud <<<"UPDATE $DB_PREFIXjobs SET reserved_at=0;"`
which did not work, so I added the closing and reopening quotes, but it could well be that `${DB_PREFIX}` also works.
Sorry for the typo. I know very well that I should always copy and paste :-)

Reply:
Yeah, if you only do `$DB_PREFIXjobs`, it will assume, that's the variable name (including `jobs`). With `${...}` that can be avoided.